src/components/App/Currencies/CurrenciesList/CurrenciesList.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FlatList } from 'react-native';
+
+import { CurrencyItem } from 'components/shared/Financialtems';
+import { getCurrencyIconUri } from 'utils/currency-icon-uri';
+
+class CurrenciesList extends Component {
+  static propTypes = {
+    currencies: PropTypes.array.isRequired,
+    filterSubstring: PropTypes.string,
+    onCurrencySelected: PropTypes.func,
+  };
+
+  getUSDPrice = ({ USD: { price } }) => price.toFixed(6);
```

Comment:
You also can use our `toFixed` func from `utils`, it just removing zeros from the right side of the number.